sources/procfs.go

```diff
@@ -72,6 +75,11 @@ type lustreHelpStruct struct {
 	helpText string
 }
 
+type statsHelpStruct struct {
```

Comment:
@roclark How about multistatParsingStruct or something to that effect? That's essentially what this does. For files that have more than one stat in them, it enables parsing the stats.